iis/assets/monitors/err.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "[IIS] Increase of not found error per second for site: {{site.name}}",
```

Comment:
not found error sounds awkward, what about 404 error?

Reply:
```suggestion
	"name": "[IIS] Increase of 404 error per second for site: {{site.name}}",
```